docs/overrides/home.html

```diff
@@ -0,0 +1,151 @@
+		        	</div>
+		        </div>
+		        <div class="feature-description-container">
+		        	<div class="feature-description last-description">
+						<p class="bullet-point">|</p><p>The page previews and PDF bookmarks allow for easy navigation of your notes</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src = "/img/Layers.gif">
+		</div>
+		
+		<div class="feature feature-right">
+		    <div class="feature-text">
+		        <h2>Enhance and accelerate your note-taking</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+						<p class="bullet-point">|</p><p>Add images and create various shapes, from circles to splines to axis</p>
+					</div>
+				</div>
+				<div class="feature-description-container">
+					<div class="feature-description last-description">
+	        			<p class="bullet-point">|</p><p>Snap objects to rectangular grid or degrees of rotation</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Create Shape.png">
+		</div>
+		
+		<div class="feature feature-left">
+		    <div class="feature-text">
+		        <h2>Get scientific</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description last-description">
+	        			<p class="bullet-point">|</p><p>Create anything from differential equations to electrical circuits or the structural formula of molecules using our built-in LaTeX editor</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Latex.png">
+		</div>
+		
+		<div class="feature feature-right">
+		    <div class="feature-text">
+		        <h2>Your notes, your way</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+	        			<p class="bullet-point">|</p><p>Customize your toolbar to create a new layout tailor-made for you</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-description-container">
+		        	<div class="feature-description last-description">
+	        			<p class="bullet-point">|</p><p>Use a plugin or create your own via the Lua programming language</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Customize Toolbars.gif">
+		</div>
+		
+		<div class="feature feature-left last-feature">
+		    <div class="feature-text">
+		        <h2>Explain every stroke</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+		        		<p class="bullet-point">|</p><p>Insert audio recording to any object in your note</p>
```

Comment:
It is not possible to insert audio recordings but to record audio while you write. Small but important difference.
```suggestion
		        		<p class="bullet-point">|</p><p>Never miss anything important and record audio while you write your notes</p>
```

Reply:
But you are inserting audio recording to objects that can then be played with the play object tool.

Reply:
Yes but you record this audio while writing. This makes it seem, as if you could select an existing audio recording and insert it using some kind of file picker.

Reply:
Ah I see. Alright.